packages/slice-machine/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,6 +1,24 @@
 import { Text } from "@prismicio/editor-ui";
-import { FC, PropsWithChildren } from "react";
+import { FC } from "react";
 
-export const Breadcrumb: FC<PropsWithChildren> = ({ children }) => {
-  return <Text color="grey11">{children}</Text>;
+import * as styles from "./Breadcrumb.css";
+
+export const Breadcrumb: FC<{
+  folder?: string;
+  page?: string;
+  separator?: string;
+}> = ({ folder, page, separator = "/" }) => {
```

Comment:
I'm not sure on the structure here:

Why not having something that just look at the children, loop over it and add the separator, you could just pass ReactNode or Link or string, anything to the children array.

Something like: https://mui.com/material-ui/react-breadcrumbs/